main.js

```diff
@@ -89,7 +89,7 @@ class SplashScreen extends Phaser.Scene {
     this.load.image("levelComplete", "assets/level_complete.png");
     this.load.image("splash", "assets/splash_screen.png");
     this.load.spritesheet("dude", "assets/dude.png", { frameWidth: 24, frameHeight: 32 });
-    this.load.spritesheet("heart", "assets/heart.png", { frameWidth: 10, frameHeight: 10 });
+    this.load.spritesheet("heart", "assets/heart.png", { frameWidth: 11, frameHeight: 10 });
```

Comment:
I didn't notice the bleeding before. Are you sure it wasn't some browser outdated cache issue? In any case, I am fine with this change :smile: